.ci/scripts/test_global_patch.py

```diff
@@ -18,52 +18,84 @@
 import subprocess
 import sys
 
-# test patching from command line
-err_code = subprocess.call(
-    [sys.executable, "-m", "sklearnex.glob", "patch_sklearn", "-a", "svc"]
-)
-assert not err_code
-from sklearn.svm import SVC, SVR
-
-assert SVC.__module__.startswith("daal4py") or SVC.__module__.startswith("sklearnex")
-assert not SVR.__module__.startswith("daal4py") and not SVR.__module__.startswith(
-    "sklearnex"
-)
-
-
-from sklearnex import patch_sklearn, unpatch_sklearn
-
-# test unpatching from command line
-err_code = subprocess.call([sys.executable, "-m", "sklearnex.glob", "unpatch_sklearn"])
-assert not err_code
-unpatch_sklearn()
-from sklearn.svm import SVC, SVR
-
-assert not SVC.__module__.startswith("daal4py") and not SVC.__module__.startswith(
-    "sklearnex"
-)
-assert not SVR.__module__.startswith("daal4py") and not SVR.__module__.startswith(
-    "sklearnex"
-)
-
-
-# test patching from function
-patch_sklearn(name=["svc"], global_patch=True)
-from sklearn.svm import SVC, SVR
-
-assert SVC.__module__.startswith("daal4py") or SVC.__module__.startswith("sklearnex")
-assert not SVR.__module__.startswith("daal4py") and not SVR.__module__.startswith(
-    "sklearnex"
-)
-
-
-# test unpatching from function
-unpatch_sklearn(global_unpatch=True)
-from sklearn.svm import SVC, SVR
-
-assert not SVC.__module__.startswith("daal4py") and not SVC.__module__.startswith(
-    "sklearnex"
-)
-assert not SVR.__module__.startswith("daal4py") and not SVR.__module__.startswith(
-    "sklearnex"
-)
+import pytest
+
+
+@pytest.fixture
+def patch_from_command_line(request):
+    err_code = subprocess.call(
+        [sys.executable, "-m", "sklearnex.glob", "patch_sklearn", "-a", "svc"]
+    )
+    assert not err_code
+
+    def unpatch_from_cmd():
+        err_code = subprocess.call(
+            [sys.executable, "-m", "sklearnex.glob", "unpatch_sklearn"]
+        )
+        assert not err_code
+
+    request.addfinalizer(unpatch_from_cmd)
+    return

```

Comment:
```suggestion
    err_code = subprocess.call(
        [sys.executable, "-m", "sklearnex.glob", "patch_sklearn", "-a", "svc"]
    )
    assert not err_code
    yield
    err_code = subprocess.call(
        [sys.executable, "-m", "sklearnex.glob", "unpatch_sklearn"]
    )
    assert not err_code
```

Thoughts on using yield instead?

Reply:
What would be the advantage of using yield here?

Reply:
Its not a must, but its the pytest recommended procedure over adding a finalizer https://docs.pytest.org/en/stable/how-to/fixtures.html#yield-fixtures-recommended

Reply:
Changed to `yield`.

Reply:
Looks like the windows job doesn't like usage of 'yield':
```
INTERNALERROR>     ^^^^^^^^^^^^^^^^^^^^^^^^^
INTERNALERROR>   File "C:\Miniconda\envs\CB\Lib\site-packages\_pytest\logging.py", line 790, in pytest_collection
INTERNALERROR>     return (yield)
INTERNALERROR>             ^^^^^

...

joblib.externals.loky.process_executor.BrokenProcessPool: A task has failed to un-serialize. Please ensure that the arguments of the function are all picklable.
```

Will try to see where that multi-processing invocation is coming from.